### ISSUE TITLE
Consider replacing `sshtunnel` with a different library or reimplement here

### ISSUE THREAD

**Issue by edgarrmondragon** [MeltanoLabs/tap-postgres#99]
Last release of `sshtunnel` was on Jan 2021.

https://pypi.org/project/sshtunnel/

**Comment by visch** [comment]
Yeah one option is just wrapping paramiko yourself but there's a lot of extra little things this library seems to have done right.

**Comment by visch** [comment]
I added https://github.com/pahaz/sshtunnel/pull/277 to the upstream which fixes their e2e github action. Hopefully they give some feedback / comments.

**Comment by edgarrmondragon** [comment]
> I added [pahaz/sshtunnel#277](https://github.com/pahaz/sshtunnel/pull/277) to the upstream which fixes their e2e github action. Hopefully they give some feedback / comments.

Thanks @visch!
